src/bot.ts

```diff
@@ -0,0 +1,38 @@
+import fs from 'node:fs';
+import dotenv from 'dotenv';
+import { Client, Collection } from 'discord.js';
+import { Command } from './interfaces/Command';
+import { log } from './utils/log';
+dotenv.config();
+
+const client = new Client({ intents: ['GUILDS', 'GUILD_MESSAGES', 'GUILD_MESSAGE_REACTIONS'] });
+
+client.once('ready', () => {
+	log({
+		message: 'Bot is ready!',
+		section: 'BOT',
+		color: 'green',
+	});
+});
+
+const botCmds = new Collection();
+const cmdsFolder = fs.readdirSync('./src/commands');
+
+for (const file of cmdsFolder) {
+	const name = file.split('.')[0];
+	const cmd = import(`./commands/${name}.js`);
```

Comment:
I would suggest removing the `.js` suffix. But it's only required if we start using ts-node.

Reply:
If you make an PR about https://github.com/palavreco/Palavreco/pull/9#discussion_r854122928, could you also test it? I made it because the actual code that we will run is `.js`. Without the extention it gave me errors.